index.js

```diff
@@ -50,8 +49,11 @@ function test(p, opts) {
   });
 }
 
+exports.test = test;
```

Comment:
I thought it would be a way to nest the results output inside a context if the user wanted, but it turns out it doesn't quite work that way (output is appended to the end)

Desired Results
```
My Mocha Context
  eslint
    ✓ should have no errors in my/file/name.js
    ...
```